examples/chem-sync-local-flask/local_app/app.py

```diff
@@ -25,7 +25,7 @@ def receive_webhooks(target: str) -> tuple[str, int]:  # noqa: ARG001
 
         # Important! To verify webhooks, we need to pass the body as an unmodified string
         # Flask's request.data is bytes, so decode to string. Passing bytes or JSON won't work
-        verify(app_def_id, request.data.decode("utf-8"), request.headers)
+        verify(app_def_id, request.data.decode("utf-8"), request.headers) # type: ignore[arg-type]
```

Comment:
it doesn't look like anything in this line changed.... did it suddenly start reporting type errors? (that's very weird and very annoying). We should ideally not have too many places where we suppress the linter like this

Reply:
Yeah, exactly. Mypy complained that it was expecting a dict but it got Headers, so I took the lazy route. But I also didn't get this error when I've made PRs previously.

Reply:
mmm I was able to reproduce this. 
It looks like this is in fact a type error. I'm uncertain why it wasn't failing already. 
We will potentially have to update the function signature for `verify` in order for the type checking here to pass. For now, we will just use this type ignore

Reply:
@keithmacaulay That type annotation is fixed as of SDK version 1.21.0.